### ISSUE TITLE
Join forces with other percona cookbook maintainers?

### ISSUE THREAD

**Issue by patcon** [customink-webops/percona-install#6]
Just realizing that there seems to be some duplicated efforts. Any thoughts on merging? I also have a modest proposal in another cookbook issue queue in regards to leveraging the newly flexible mysql cookbook as a dependecy: phlipper/chef-percona#15

Cheers!

**Comment by phlipper** [comment]
@patcon which cookbook(s) specifically are you looking at? I'd be willing to consider merging efforts if there was a clear case to do so.

Thanks.

**Comment by patcon** [comment]
Not sure of the clear case aside from the fact that we're all publicly sharing 3 different reasonably complete cookbooks, and any one currently represents a risk for others to start using, as there's a chance it won't the the reigning champ :)

I'm indifferent about which cookbook is used a starting point, as you're both equally active maintainers, and it seems your network graphs are comparable (you have opensaas and he has scribd, but he _does_ have more general forks...). Either one of you have any particular reason that you'd resist using the other's? If so, that would be the only reason I'd lean one way or the other!

//cc @dje @heavywater

**Comment by phlipper** [comment]
@patcon sorry, I thought this was attached to my repo when I initially commented. I would be happy to combine efforts with with existing cookbooks. My personal commitment timeline will depend on how much work is involved, but I would be happy to see a single "canonical" resource for the percona cookbook.

**Comment by nathenharvey** [comment]
I agree that it makes sense to come up with a canonical cookbook for installing / configuring Percona and further that we should try to leverage the existing MySQL cookbook so that it can be used to install the percona version of MySQL.

Next steps?

Should we try a quick synchronous conversation about the matter via IRC, skype, g+ hangout, etc.?

**Comment by patcon** [comment]
I'm down. @phlipper is on the west coast, so how about 4pm EST Wednesday (tomorrow) on skype? My skype is patrick.c.connolly and I'm on it all day for work. But I'm easy if something else is simpler.

**Comment by nathenharvey** [comment]
I'm nathenharvey on skype.  4PM today would work.

**Comment by phlipper** [comment]
Hey guys, 4pm EST works for me. I'll ping you around then. Thanks!

**Comment by nathenharvey** [comment]
I'm nathenharvey on Skype.

## 

Nathen Harvey

On Sep 5, 2012, at 12:48 PM, Phil Cohen notifications@github.com wrote:

Hey guys, 4pm EST works for me. I'll ping you around then. Thanks!

—
Reply to this email directly or view it on
GitHubhttps://github.com/customink-webops/percona-install/issues/6#issuecomment-8305454.
